### ISSUE TITLE
IOS (Swift) Sample with new speech SDK

### ISSUE THREAD

**Issue by ukanth** [Azure-Samples/cognitive-services-speech-sdk#62]
Please update sample for iOS (swift) with new speech SDK. 

### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [ ] feature request
- [x] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
>

### Any log messages given by the failure
>

### Expected/desired behavior
>

### OS and Version?
> Swift/MacOS/XCode

### Versions
>

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by chlandsi** [comment]
There aren't any Swift bindings of the SDK yet, although they are being planned for one of the next relases. However, [iOS bindings for Objective-C](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/quickstart-objectivec-ios) are available.

**Comment by rohitgadia** [comment]
Hi,

We are planning to use the Objective-C SDK in our Swift based application via the Bridging Header, but we are facing an issue when we try to leverage the Microphone Streaming as there's no valid sample app to refer to, although the quickstart example that you have mentioned in the reply only takes a file input (wav).

Is the integration of Objective-C SDK via Bridging Header and using Microphone Stream to perform Speech to Text not possible currently? 

If it's possible would you mind sharing with us a sample for the same?

**Comment by chlandsi** [comment]
Could you give some details on the issue you are encountering?

We will update the quickstart article to cover usage of the microphone soon.

**Comment by ukanth** [comment]
Thanks @chlandsi. We are looking for microphone specific examples (ObjC/Swift) Please let us know once updated.

**Comment by rohitgadia** [comment]
Any ETA on this microphone specific example's availability?

**Comment by chlandsi** [comment]
@rohitgadia it should be ready by next week or so. In Objective-C, this snippet can be adapted to do speech recognition from the microphone:

```objc
    SPXSpeechConfiguration *speechConfig = [[SPXSpeechConfiguration alloc] initWithSubscription:speechKey region:serviceRegion];
    if (!speechConfig) {
        NSLog(@"Could not load speech config");
        return;
    }
        
    SPXSpeechRecognizer* speechRecognizer = [[SPXSpeechRecognizer alloc] init:speechConfig];
    if (!speechRecognizer) {
        NSLog(@"Could not create speech recognizer");
        return;
    }

```

**Comment by chlandsi** [comment]
The updated quickstart tutorial for Objective-C is [now available](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/quickstart-objectivec-ios). Note that we also released a new version of the SDK (1.0.1) available from https://aka.ms/csspeech/iosbinary, which includes a few fixes.

Closing this issue -- if there are problems using the sample, please open a new one.
